content/v1/csidriver/installation/helm/unity.md

```diff
@@ -17,49 +17,98 @@ The controller section of the Helm chart installs the following components in a
 
 The node section of the Helm chart installs the following component in a _DaemonSet_:
 
-- CSI Driver for Unity
+- CSI Driver for Unity XT
 - Kubernetes Node Registrar, which handles the driver registration
 
 ## Prerequisites
 
-Before you install CSI Driver for Unity, verify the requirements that are mentioned in this topic are installed and configured.
+Before you install CSI Driver for Unity XT, verify the requirements that are mentioned in this topic are installed and configured.
 
 ### Requirements
 
 * Install Kubernetes or OpenShift (see [supported versions](../../../../csidriver/#features-and-capabilities))
 * Install Helm v3
-* To use FC protocol, the host must be zoned with Unity array and Multipath needs to be configured
+* To use FC protocol, the host must be zoned with Unity XT array and Multipath needs to be configured
 * To use iSCSI protocol, iSCSI initiator utils packages needs to be installed and Multipath needs to be configured 
 * To use NFS protocol, NFS utility packages needs to be installed
 * Mount propagation is enabled on container runtime that is being used
 
+### Install Helm 3.0
+
+Install Helm 3.0 on the master node before you install the CSI Driver for Dell Unity XT.
+
+**Steps**
+
+Run the `curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash` command to install Helm 3.0.
+
+
+### Fibre Channel requirements
+
+Dell Unity XT supports Fibre Channel communication. If you use the Fibre Channel protocol, ensure that the
+following requirement is met before you install the CSI Driver for Dell Unity XT:
+- Zoning of the Host Bus Adapters (HBAs) to the Fibre Channel port must be done.
+
+
+### Set up the iSCSI Initiator
+The CSI Driver for Dell Unity XT supports iSCSI connectivity.
+
+If you use the iSCSI protocol, set up the iSCSI initiators as follows:
```

Comment:
Change sentence to read "If you use the iSCSI protocol, follow this procedure to set up the iSCSI initiators: